go.mod

```diff
@@ -60,6 +60,7 @@ replace (
 	// controller runtime
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200331152225-585af27e34fd // release-4.5
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
+	github.com/operator-framework/api => /Users/jhunkins/go/src/github.com/operator-framework/api
```

Comment:
Note to other reviewers - we're waiting on https://github.com/operator-framework/api/pull/139 to merge before this pin can be removed.

Reply:
api reference has been updated